.changes/tinymce/7.7.0.md

```diff
@@ -0,0 +1,27 @@
+## 7.7.0 - 2025-02-17
+
+### Added
+- `link_attributes_postprocess` option that allows overriding attributes of a link that would be inserted through the link dialog. #TINY-11707
+
+### Improved
+- Improved visual indication of a keyboard focus for a comment annotation when there is an image inside. #TINY-11596
+- Not specifying a notification type (or specifying incorrect one) now defaults to 'info'. #TINY-11661
+- Improved visual separation of comments side panel header. #TINY-11715
```

Comment:
This belongs in the premium changelog, even though it was a core fix. It won't impact open source users.

- [ ] move to the premium changelog